docs/book/v1/authorization-server.md

```diff
@@ -6,6 +6,8 @@ for your application.
 Since there are authorization flows that require user interaction,
 **your application is expected to provide the middleware to handle this**.
 
+To enable the authorization server to parse the requests from JSON, the [Body Parsing Middleware](https://docs.mezzio.dev/mezzio/v3/features/helpers/body-parse/) helper should be included in the application. 
+
```

Comment:
The current page contains the following steps:

* [Add the token endpoint](https://docs.mezzio.dev/mezzio-authentication-oauth2/v1/authorization-server/#add-the-token-endpoint)
* [Add the authorization endpoint](https://docs.mezzio.dev/mezzio-authentication-oauth2/v1/authorization-server/#add-the-authorization-endpoint)

And if the body parsing middleware is needed here, it should be added as a separate step under its own heading.